Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```
@@ -10,6 +10,8 @@
     allowedModes:
     - HyposprayDynamicMode
     - HyposprayInjectMode
+  - type: Spillable
+    solution: hypospray
   - type: Appearance
```

Add another component to make the solution examinable - I believe it's ExaminableSolution?
```suggestion
  - type: ExaminableSolution
    solution: hypospray
  - type: Appearance
```